hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,8 @@
+RELEASE_TYPE: minor
+
+This release limits the maximum duration of the shrinking phase to five minutes,
+so that Hypothesis never appears to hang or enter an infinite loop when we are
```

Comment:
Nitpicking over phrasing:

I'd delete "or enter an infinite loop" (it makes it look like we can actually enter an infinite loop).

Also Hypothesis can still appear to hang if your test can hang, so I'd maybe delete "never" as well. Something like "so that Hypothesis does not appear to hang when..."